helm/cert-exporter-chart/templates/daemonset.yaml

```diff
@@ -26,9 +26,17 @@ spec:
       - name: cert-exporter
         image: quay.io/giantswarm/cert-exporter:[[ .SHA ]]
         args:
+        {{ if (.Values.Installation) }}
```

Comment:
Annoyingly, Helm seems to choke if we do something like
```
{{ if (.Values.Installation.V1.Monitoring.CertExporter.CertPath) }}
```
assumedly because we're trying to check if V1 exists under a non-existent key.
This would arguably be safer, but the overall installations schema is pretty solid anyway.

Reply:
Is this because of the difference between control plane and tenant cluster? Don't we have a setting that tells the difference already so we can use this specific check then?

Reply:
I guess this would be that check? The only real difference at installation is the helm values file.